src/codegen/forward_demand.jl

```diff
@@ -271,11 +286,12 @@ function forward_diff_no_inf!(ir::IRCode, interp, mi::MethodInstance, world, to_
             end
         end
     end
-
 end
 
+Base.@deprecate forward_diff_no_inf!(ir::IRCode, interp, mi::MethodInstance, world, to_diff::Vector{Pair{SSAValue, Int}}; kwargs...) forward_diff_no_inf!(ir, to_diff; kwargs...)
```

Comment:
Do we need this?
Given the package is currently unregistered.
And this is an internal method.

OTOH i know we have some internal code that calls this method directly

Reply:
Not required

Reply:
```suggestion
```